variables.tf

```diff
@@ -11,50 +11,45 @@ variable "subnet_ids" {
   type        = list(string)
   description = "List of subnet IDs for the targets of the ALB."
 }
-variable "default_target_group_config" {
-  type = object({
-    type                       = string // instance
-    deregistration_delay       = number // 300
-    slow_start                 = number // 0
-    stickiness_cookie_duration = number // null
+
+variable "target_groups" {
+  type = map(object({
+    port                 = number
+    type                 = string
+    deregistration_delay = number
+    // 300
+    slow_start = number
+    // 0
```

Comment:
You might want to nuke these random comments.

Reply:
done